### DISCUSSION TITLE
Encrypting multiple files or folders?

### DISCUSSION THREAD

**Discussion by Chippel** [FiloSottile/age#439] [Q&A]
I just started using age. It seems like it's way faster and way more convenient than using PGP. Unfortunately it seems like it is not possible to encrypt multiple files and or folders? Is something like encrypting multiple files planned?

**Comment by FiloSottile** [comment]
How does the CLI look like for encrypting multiple files with gpg?

**Reply by Chippel** [reply]
I'm kinda confused by your reply. It seems like GPG can encrypt multiple files by using the gpgtar tool which is installed by default.
I like age more tho. It uses way more secure crypto in my opinion. The only thing I'm missing is encrypting multiple files natively. Are there any plans about bringing that feature in the future?

**Comment by FiloSottile** [comment]
I simply didn’t know about gpgtar! I’ll have a look.

Myself, I encrypt multiple files by piping tar into age. age is designed to work well as a composable UNIX tool, and it’s secure to use it like that.
